Derived from https://github.com/CachyOS/kernel-patches/blob/master/6.14/misc/nvidia/0003-drm-Remove-driver-data-from-struct-drm_driver.patch

This was removed in 6.14 from commit cb2e1c2136
("drm: remove driver date from struct drm_driver and all drivers")

diff --git a/nvidia-drm/nvidia-drm-drv.c b/nvidia-drm/nvidia-drm-drv.c
index 4280d1e..256fa11 100644
--- a/nvidia-drm/nvidia-drm-drv.c
+++ b/nvidia-drm/nvidia-drm-drv.c
@@ -61,6 +61,7 @@
 #endif
 
 #include <linux/pci.h>
+#include <linux/version.h>
 
 /*
  * Commit fcd70cd36b9b ("drm: Split out drm_probe_helper.h")
@@ -908,7 +909,10 @@ static struct drm_driver nv_drm_driver = {
     .name                   = "nvidia-drm",
 
     .desc                   = "NVIDIA DRM driver",
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 14, 0)
     .date                   = "20160202",
+#endif
 
 #if defined(NV_DRM_DRIVER_HAS_DEVICE_LIST)
     .device_list            = LIST_HEAD_INIT(nv_drm_driver.device_list),
